app/queries/get_embedded_content.rb

```diff
@@ -0,0 +1,44 @@
+module Queries
+  class GetEmbeddedContent
+    attr_reader :target_content_id, :states
+
+    def initialize(target_content_id)
+      self.target_content_id = target_content_id
+      self.states = %i[draft published]
```

Comment:
I haven't gone as far as making these fully configurable as I can't verify whether we actually need it! That said this is hopefully where it can be injected in from a query param in future 🤞